apps/qubit/modules/clipboard/actions/exportAction.class.php

```diff
@@ -516,23 +516,17 @@ private function getJobNameString()
         {
           return 'arInformationObjectCsvExportJob';
         }
-        else
-        {
+
           return 'arInformationObjectXmlExportJob';
```

Comment:
Broken indentation from this commit is fixed later.

Reply:
🤯 😆